pkg/spec/spec.go

```diff
@@ -203,3 +204,11 @@ func escapeString(s string, preventEscapingPrefix string) string {
 		return strings.Repeat("\\", (len(match)/2)-1) + "$" + preventEscapingPrefix
 	})
 }
+
+var escapeFleetVarPattern = regexp.MustCompile(`(\$FLEET_VAR_\w+)|(\${FLEET_VAR_\w+})`)
+
+func escapeFleetVar(s string, preventEscapingPrefix string) string {
```

Comment:
Just wondering what that's for? It's so that `$FLEET_VAR_...` get passed on to the server as-is as it's only on the server that we expand them?

Reply:
Yes. This is so we can put things like `$FLEET_VAR_HOSTNAME` in profiles that are uploaded via gitops. And they are expanded by the server at the right time.